packages/server/src/arpa_reporter/services/generate-arpa-report.js

```diff
@@ -925,6 +946,7 @@ async function setCSVData(data) {
 
     // ignore empty CSV files
     if (csvData.length === 0) {
+        logger.info(`No data for ${name} CSV`);
```

Comment:
One of the main reasons to use structured logging is the ability to keep the main message free from variables, which allows for much easier querying. While I won't say there's _never_ a valid reason to log an interpolated message, any time it happens is usually a good opportunity to think about whether it could be moved to a field on the logged output instead. 

In this particular instance, I think it seems worth moving to a standalone field:
```js
        logger.info({ csv: { name } }, 'no data for csv');
```

Note: I struggle to remember this too, but it's also useful to to avoid sentence-casing (i.e. starting all log messages with an uppercase letter) unless the capitalization is super significant, since it's ultimately yet another detail you'll need to keep in mind when querying log data.

Reply:
**UPDATE:** I see below that the `logger` arguments passed to this function are already configured with a `csvObj` field. Given that, simply omitting the variable from the string would be better:
```suggestion
        logger.info('no data for csv');
```

Reply:
Also, in case you're wondering what querying for these logs would look like, a few examples:

- In Datadog: 
  - `@csvObj.name:"project2128BulkUpload" "no data for csv"`
  - `@csvObj.name:("project2128BulkUpload" OR "project236BulkUpload") "no data for csv"`
- In CloudWatch: 
  - `{ ($.csvObj.name = "project2128BulkUpload") && ($.msg = "no data for csv") }`
  - `{ ($.csvObj.name = "project2128BulkUpload" || $.csvObj.name = "project236BulkUpload" ) && ($.msg = "no data for csv") }`